backend/associations/serializers/association.py

```diff
@@ -1,43 +1,23 @@
 from rest_framework import serializers
 
-from associations.models import Association, Media
-from associations.serializers.media import MediaSerializer
-from associations.serializers.page import PageShortSerializer
+from associations.models import Association
 
 
-from associations.serializers.marketplace import MarketplaceShortSerializer
-from associations.serializers.library_short import LibraryShortSerializer
 from associations.serializers.role import RoleSerializer
 
 
 class AssociationSerializer(serializers.ModelSerializer):
-    pages = serializers.SerializerMethodField()
-    marketplace = MarketplaceShortSerializer(read_only=True)
-    library = LibraryShortSerializer(read_only=True)
     my_role = serializers.SerializerMethodField(read_only=True)
 
     class Meta:
         model = Association
-        read_only_fields = ("pages", "marketplace", "library", "my_role")
-        fields = read_only_fields + ("id", "name", "logo", "is_hidden", "rank")
+        read_only_fields = ("id", "my_role", "marketplace_enabled", "library_enabled")
```

Comment:
Maybe put these properties in a "enabled" field which contains marketplace and library, to add other functionalities in the futur (such  as medias for instance)